ruby/.gitignore

```diff
@@ -1,3 +1,4 @@
+test/fixtures/log/junit.xml
```

Comment:
DId you mean to ignore this? It looks like it's used in the tests below.

Reply:
> Would this conflict with the reporter we currently use, or would the operate independently?

They'd work concurrently, however if they both try to write in the same path, one will win over the other. But there won't be name clashes or anything like that.

Reply:
Yes, I want to ignore it because to written by the integration tests.